cyber/BUILD

```diff
@@ -7,7 +7,7 @@ cc_library(
     name = "cyber",
     linkstatic = False,
     deps = [
-        "//cyber:cyber_core",
+        ":init",
```

Comment:
Better find a way to keep the original deps targets while add the functionality in your PR.
`:init` seems like a local target name which should not have public visibility. ( I must admit that apollo has poor visibility control for now...)

Reply:
I also want to not change so many BUILD files. Do you have better ideas on how to make it simpler? I can change it accordingly.

Reply:
> I also want to not change so many BUILD files. Do you have better ideas on how to make it simpler? I can change it accordingly.

Maybe,  fine-grained header includes ? so that `:init` can depend on specific targets rather than `cyber_core`.
I will work on fixing this  with you. Hopes we can resolve it before next Monday.